modules/deployment-plug-in-cluster.adoc

```diff
@@ -10,32 +10,79 @@ After pushing an image with your changes to a registry, you can deploy the plug-
 
 .Procedure
 
-. To deploy your plug-in to a cluster, instantiate your plug-in by running the following command:
+. To deploy your plug-in to a cluster, install a Helm chart with the name of the plug-in as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option. Provide the location of the image within the `plugin.image` parameter by using the following command:
+
 +
 [source,terminal]
 ----
-$ oc process -f template.yaml \
-  -p PLUGIN_NAME=my-plugin \ <1>
-  -p NAMESPACE=my-plugin-namespace \ <2>
-  -p IMAGE=quay.io/my-repository/my-plugin:latest \ <3>
-  | oc create -f -
+helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
 ----
-<1> Update with the name of your plug-in.
-<2> Update with the namespace.
-<3> Update with the name of the image you created.
-+
-This command runs a light-weight NGINX HTTP server to serve the assets for your plug-in.
 
-[IMPORTANT]
-====
-`PLUGIN_NAME` must match the plug-in name you used in the `consolePlugin` declaration of `package.json`.
-====
+. Optional: You can specify additional parameters by using the values set of supported parameters. 
```

Comment:
How to use following YAML as optional deploy method?

Reply:
I added the file location that lists the optional parameters